### PR TITLE
Repositioned "read more" links

### PR DESCRIPTION
**Pull request by lindaicing**
Repositioned "read more" links
Reformatted menu navigation
Screen Shot 2020-06-18 at 12 34 48 AM

### CODE REVIEW

File: Website-and-Backend/src/main/resources/public/stylescripts/script.js [hunk 1]
```diff
@@ -56,19 +56,20 @@ $(document).ready(function() {
...
+            })
+        }else{
             $("#navlinks").removeClass("navClicked");
         }
+    }
+
+    $(window).resize(function(){
+        mobileNav();
     })
+    mobileNav();
```
Comment (CubeOfCheese) [inline Website-and-Backend/src/main/resources/public/stylescripts/script.js:72]: What is this?

Comment (CubeOfCheese) [reply Website-and-Backend/src/main/resources/public/stylescripts/script.js:72]: Like why is it called on line 72? It seems like it is meant to only be called for mobile devices right?

Comment (lindaicing) [reply Website-and-Backend/src/main/resources/public/stylescripts/script.js:72]: Yes, the code is called in two places: both at mobile sizes. Once when the window loads, and the other when the window resizes

Comment (lindaicing) [reply Website-and-Backend/src/main/resources/public/stylescripts/script.js:72]: It's meant to make sure the menu toggles the links properly. Without the code in both places, it won't work (before or after resizing the page)

Comment (CubeOfCheese) [reply Website-and-Backend/src/main/resources/public/stylescripts/script.js:72]: Could you add a comment explaining the purpose of the function and/or where it is being called for documentation purposes?

**Pull request merged**